packages/elastic-sip-trunking/README.md

```diff
@@ -0,0 +1,121 @@
+# Sinch Elastic SIP Trunking SDK for Node.js
+
+This package contains the Sinch Elastic SIP Trunking SDK for Node.js for use with [Sinch APIs](https://developers.sinch.com/). To use it, you will need a Sinch account. Please [sign up](https://dashboard.sinch.com/signup) or [log in](https://dashboard.sinch.com/login) if you already have one.
+
+## Installation
+
+We recommend to use this SDK as part of the `@sinch/sdk-core` package as it will take care about the authentication plugins to use.
+
+However, it's still possible to use this SDK standalone is you need to access the Elastic SIP Trunking API only.
+
+### With NPM
+
+```bash
+npm install @sinch/elastic-sip-trunking
+```
+
+### With Yarn
+
+```bash
+yarn add @sinch/elastic-sip-trunking
+```
+
+## Usage
+
+### Credentials
+
+The `Elastic SIP Trunking` API uses the Sinch unified authentication with OAuth2. You will need to provide the following credentials:
+- projectId: can be found in the [Account Dashboard](https://dashboard.sinch.com/settings/access-keys)
+- keyId:: can be found in your Access key list in the [Account Dashboard](https://dashboard.sinch.com/settings/access-keys)
+- keySecret: can be found **ONLY** when generating a new access key: keep it safe!
+
+### As part of the Sinch SDK
```

Comment:
Indeed, now that I promote the usage of the service instead of the Sinch Client, it really makes sense to factorize the documentation.
Thanks for the suggestion, I'll do it for all the APIs in a separate PR